src/components/AssetHeader/AssetMarketData.tsx

```diff
@@ -104,7 +104,11 @@ export const AssetMarketData: React.FC<AssetMarketDataProps> = ({ assetId }) =>
                 <Text translation='assets.assetDetails.assetHeader.maxTotalSupply' />
               </StatLabel>
               <StatValue isLoaded={isLoaded}>
-                <Amount.Supply value={marketData?.maxSupply ?? 0} />
+                <Amount
+                  value={marketData?.maxSupply ?? 0}
+                  abbreviated={true}
+                  omitDecimalTrailingZeros={true}
```

Comment:
Just a syntactic nit but I find the shorthand true syntax to be more declarative

```suggestion
                  abbreviated
                  omitDecimalTrailingZeros
```

(No need to commit the suggestion as it will fail lint and really doesn't matter)